rollup.config.js

```diff
@@ -15,6 +17,9 @@ export default [{
   plugins: [
     json(),
     resolve(),
+    replace({
+      'PACKAGE_VERSION_INJECTED_BY_ROLLUP_DURING_BUILD': version
```

Comment:
This correctly addresses the `rollup` build process, however we're also using `tsc` to create the ESM files.

The value `PACKAGE_VERSION_INJECTED_BY_ROLLUP_DURING_BUILD` does not get replaced on build time in this file `/js-url-gen/dist/internal/utils/packageVersion.js`, the tests miss this because the file is mocked.

Maybe we can run add a new test that runs on deployment that builds the project first, then tests on the `./dist` file?

Reply:
Fixed with replaceStrings.ts